packages/contracts/conf/config.json

```diff
@@ -2,27 +2,30 @@
   "paramDefaults": {
     "minDeposit": 10,
     "pMinDeposit": 100,
-    "applyStageLength": 120,
+    "applyStageLength": 18000,
```

Comment:
Just curious, what is the reason for the bump in these values?

Reply:
these are just test values for now, 120 was way too short to do real testing so I bumped up the values (by doing reparameterization proposals) and changed values here so they don't get reverted when we deploy a new version